lib/commons/utils/valid-html-tags.js

```diff
@@ -0,0 +1,134 @@
+const htmlTags = [
+	'a',
+	'abbr',
+	'address',
+	'area',
+	'article',
+	'aside',
+	'audio',
+	'b',
+	'base',
+	'bdi',
+	'bdo',
+	'blockquote',
+	'body',
+	'br',
+	'button',
+	'canvas',
+	'caption',
+	'cite',
+	'code',
+	'col',
+	'colgroup',
+
+	'data',
+	'datalist',
+	'dd',
+	'del',
+	'details',
+	'dfn',
+	'dialog',
+	'div',
+	'dl',
+	'dt',
+	'em',
+	'embed',
+	'fieldset',
+	'figcaption',
+	'figure',
+	'footer',
+	'form',
+	'h1',
+	'h2',
+	'h3',
+	'h4',
+	'h5',
+	'h6',
+	'head',
+	'header',
+	'hgroup',
+	'hr',
+	'html',
+	'i',
+	'iframe',
+	'img',
+	'input',
+	'ins',
+	'kbd',
+	'keygen',
+	'label',
+	'legend',
+	'li',
+	'link',
+	'main',
+	'map',
+	'mark',
+	'math',
+	'menu',
+	'menuitem',
+	'meta',
+	'meter',
+	'nav',
+	'noscript',
+	'object',
+	'ol',
+	'optgroup',
+	'option',
+	'output',
+	'p',
+	'param',
+	'picture',
+	'pre',
+	'progress',
+	'q',
+	'rb',
+	'rp',
+	'rt',
+	'rtc',
+	'ruby',
+	's',
+	'samp',
+	'script',
+	'section',
+	'select',
+	'slot',
+	'small',
+	'source',
+	'span',
+	'strong',
+	'style',
+	'sub',
+	'summary',
+	'sup',
+	'svg',
+	'table',
+	'tbody',
+	'td',
+	'template',
+	'textarea',
+	'tfoot',
+	'th',
+	'thead',
+	'time',
+	'title',
+	'tr',
+	'track',
+	'u',
+	'ul',
+	'var',
+	'video',
+	'wbr'
+];
+
+/**
+ * Verifies that if a given html tag is valid
+ * @method isValidHtmlTag
+ * @memberof axe.commons.utils
+ * @instance
+ * @param htmlTag htmlTag to check if valid
+ * @return {Boolean} true/ false
+ */
+axe.utils.isValidHtmlTag = function isValidHtmlTag(htmlTag) {
+	'use strict';
+	return htmlTags.includes(htmlTag.toLowerCase());
```

Comment:
This will match elements of other markup languages too. For example, SVG has A, SCRIPT and STYLE elements. That could lead to false positives.